src/jit/lsra.cpp

```diff
@@ -1868,6 +1868,14 @@ void LinearScan::identifyCandidates()
             continue;
         }
 
+#if FEATURE_MULTIREG_ARGS_OR_RET        
+        if (varDsc->lvIsMultiRegArgOrRet)
+        {
+            varDsc->lvLRACandidate = 0;
+            continue;
+        }        
+#endif
+
```

Comment:
@sivarv - your understanding is correct, but there are multiple places in importer where 'lvIsMultiRegArgOrRet` is set, and only one of them sets `lvDoNotEnregister' (and it sets it directly, instead of calling `lvaSetVarDoNotEnregister` as I believe it should be doing). Furthermore, it is checked directly in `Lowering::TreeNodeInfoInit()` where it sets a return value to be contained. Also, we don't need/want to limit this to SIMD because 1) any other structs will not currently reach this point, and 2) when/if we decide to fully promote other structs, those would be impacted as well until such time as we fix this - including `treeNodeInfoInit()`.